mslib/msui/multilayers.py

```diff
@@ -0,0 +1,470 @@
+from PyQt5 import QtWidgets, QtCore
```

Comment:
add the standard header of the project. Add your name: into the line :copyright: Copyright 2021 May Bär